index.html

```diff
@@ -203,17 +208,31 @@
     <section><h4>Exposing a Thing</h4>
     <ul>
       <li>
-        Exposing the <a>Thing</a> includes generating the protocol bindings in order to access lower level functionality.
+        Exposing the <a>Thing</a> includes generating the protocol bindings in
+        order to access lower level functionality.
+      </li>
+      <li>
+        Create a local <a>Thing</a> to be exposed, based on <a>Thing Description</a>.
       </li>
       <li>
-        Create a local <a>Thing</a> to be exposed, based on a <a>Thing Description</a> provided in string serialized format, or out of an existing <a>Thing</a> object.
+        The following use cases can be implemented before creating the
+        <a>Thing</a> by editing the <a>Thing Description</a>:
+        <ul>
+          <li>Add a <a>Property</a> definition to the <a>Thing</a>.</li>
+          <li>Remove a <a>Property</a> definition from the <a>Thing</a>.</li>
+          <li>Add an <a>Action</a> definition to the <a>Thing</a>.</li>
+          <li>Remove an <a>Action</a> definition from the <a>Thing</a>.</li>
+          <li>Add a WoT <a>Event</a> definition to the <a>Thing</a>.</li>
+          <li>Remove a WoT <a>Event</a> definition from the <a>Thing</a>.</li>
+        </ul>
+        <p class="ednote">
+          After evaluating dynamic modifications to <a>Thing Descriptions</a>
+          through several versions of this API, the editors concluded that the
+          simplest way to represent these use cases it to take an existing
```

Comment:
Missing *is*: *...these use cases it to...* ->  *...these use cases it IS to...*

Reply:
Thanks for spotting. I corrected and will squash the commit in the first one before merging.